flint.ui/src/views/flint/ConfigurationsPoint.vue

```diff
@@ -122,6 +143,19 @@ export default {
       this.$store.commit('set_pointConfig_Pool_3', this.pool3.value.toFixed(1))
 
       this.$store.dispatch('send_pointConfig')
+    },
+
+    Run(){
+      this.$root.$refs.finalPoolValues(); //This does whatever the stepper does.
+
+      //If the setup is ok it will take the user to the Output else give an alert.
+      if(this.$store.state.point.flag===1){  
+        this.$store.dispatch('parse_point_results')
+        window.open('http://127.0.0.1:8000/flint/point_output_table',"_self");
```

Comment:
Instead of using `window.open`, try navigating to the outputs page with the help of `vue-router`. Plus we can omit the absolute url path.

Reply:
On it! ✌️